_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,33 @@
+This interest in being part of that is really the paramount concern.
+There are lots of great engineers out there, all with their own styles and preferred ways of working in a team.
+We really wanted to focus on finding team members who would be adding to and strengthening the values we have in place.
+
+After talking about technical challenges, and how we might tailor something to be most useful to what we needed,
+we coalesced on pair programming. We do not practice strict formal pairing here as a whole,
+but we really like to collaborate, especially on large features.
+It is really common for each person to either hop on a call with another engineer about a challenge, an idea, or to talk over code review. Settling on that as the common factor, being able to communicate well with any new person, was really key.
+
+In the end we set up a process for pairing with a candidate that could be done in about an hour, and had both a back-end and front-end section (this was important for us because as a team, we are all full-stack).
+We choose to go with a three person call, one teammate leading the frontend, and another teammate to lead the backend portion. This worked out well for us because then two engineers get to talk to the candidate, and the candidate also gets to ask questions and see how we work with each other. The idea is to give a small window into how we work, while also getting an idea about how the candidate might fit into that with us. Our setup is based on some cloned codebases of ours with a few features removed. This worked for us because it was nice to see a candidate work with code that is based on our real codebase, and also makes it a little simpler for us keep in mind some of ways to fill in the missing code.
+
+Once we start the exercise with the candidate, the main point is to have them be the ones driving the solutions, and communicating with us about what they are thinking. We try to make it clear, that we want to see how they work. We try to really stress that it is not a test. We are completely happy with talking it out, looking at docs for syntax and sudo-coding solutions as a first pass. Also, it is great to hear about optimizations the candidate might make, or what they may change if going about it as a task with more time to complete. Overall we this whole set up lets us get a sense for a person’s personality, disposition and problem solving style. I have to say I have really enjoyed getting a chance to talk to so many engineers, and  see how they communicate. All in all, the goal is to make it as painless for the candidates as possible while giving us all the chance to get to know each other.
+
+This is an energy intensive process for us for sure, but so far, getting to the opportunity to get to know people, and giving everyone a chance to really show where they may fit in a team has been a really a good investment for us. The team is growing, and it feels like it is growing in all the right ways.
```

Comment:
• energy-intensive

• ~getting to~ the opportunity ~to get~ to ~know~ `meet` people ~,~ (no comma) and ~giving everyone a chance to really~ show`ing` where they may fit ~in a~ `with the` team has been a really a good investment for us.

Reply:
🍍